client/js/client-ui.js

```diff
@@ -310,6 +317,8 @@ class BaseUI {
       } else {
         this.input_index = this.input_history.length;
       }
+      //TODO parse irc commands here
+      this.client.send(hostId, serverName, text);
```

Comment:
This looks like it will send the raw text, as well as parse the IRC command (lines 324-325). I think maybe this is a bad merge and these lines should be removed.